src/SwedbankPay.Sdk.Tests/Json/CustomRegionInfoConverterTests.cs

```diff
@@ -1,51 +1,53 @@
-﻿using System;
-using System.Globalization;
-
-using Newtonsoft.Json;
-using Newtonsoft.Json.Linq;
-
-using Xunit;
-
-namespace SwedbankPay.Sdk.Tests.Json
-{
-    public class CustomRegionInfoConverterTests
-    {
-        [Theory]
-        [InlineData("SE")]
-        [InlineData("NO")]
-        [InlineData("GB")]
-        public void CanDeSerialize_RegionInfo(string regionInfoString)
-        {
-            //ARRANGE
-            var jsonObject = new JObject { { "region", regionInfoString } };
-
-            //ACT
-            var result = JsonConvert.DeserializeObject<RegionInfo>(jsonObject.ToString(), JsonSerialization.JsonSerialization.Settings);
-
-            //ASSERT
-            Assert.Equal(regionInfoString, result.Name);
-        }
-
-
-        [Theory]
-        [InlineData("SE")]
-        [InlineData("NO")]
-        [InlineData("GB")]
-        public void CanSerialize_RegionInfo(string regionInfoString)
-        {
-            //ARRANGE
-            var dummy = new
-            {
-                Region = new RegionInfo(regionInfoString)
-            };
-
-            //ACT
-            var result = JsonConvert.SerializeObject(dummy, JsonSerialization.JsonSerialization.Settings);
-            var obj = JObject.Parse(result);
-
-            obj.TryGetValue("Region", StringComparison.InvariantCultureIgnoreCase, out var region);
-            //ASSERT
-            Assert.Equal(regionInfoString, region);
-        }
-    }
+﻿using System.Globalization;
+using System.Text.Json;
+
+using Xunit;
+
+namespace SwedbankPay.Sdk.Tests.Json
+{
+    public class CustomRegionInfoConverterTests
+    {
+        [Theory]
+        [InlineData("SE")]
+        [InlineData("NO")]
+        [InlineData("GB")]
+        public void CanDeSerialize_RegionInfo(string regionInfoString)
+        {
+            //ARRANGE
+            var jsonObject = $"{{ \"region\": \"{regionInfoString}\"}}";
+
+            //ACT
+            var result = JsonSerializer.Deserialize<RegionInfo>(jsonObject.ToString(), JsonSerialization.JsonSerialization.Settings);

```

Comment:
Codacy found an issue: [There's no need to call 'ToString()' on a string.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=6290774)